docs/jwst/extract_1d/arguments.rst

```diff
@@ -45,59 +72,28 @@ The ``extract_1d`` step has the following step-specific arguments.
   0 for that particular column (or row). If "bkg_fit" is not "poly", this
   parameter will be ignored.
 
-``--bkg_sigma_clip``
```

Comment:
So is the bkg_sigma_clip option only for IFU data. I see that it is removed here and added in the IFU section

Reply:
Yes, it's only used for IFU.